### ISSUE TITLE
GLES 3.1 or Mobile Vulkan renderer

### ISSUE THREAD

**Issue by motorsep** [KozGit/DOOM-3-BFG-VR#281]
I am surprised no one to the date implemented ES3.1 or/and mobile Vulkan renderer for Doom 3 BFG VR.

Imagine running this on Oculus Go / Gear VR or Lenovo Mirage / Vive Focus ?!?!

**Comment by KozGit** [comment]
I think we're all trying to actually finish the vr version before adding alternate device support. :)   I'm actually sort of curious how the experience would be on those devices - I think I would miss motion controller support too much. 

Hopefully Vulkan will come when RB finishes merging it into RBDoom.

**Comment by CarlKenner** [comment]
Currently we're working on Go/GearVR support via ALVR streaming it from the PC. I haven't gotten ALVR to work on my GearVR yet though. There's another issue on that topic.

**Comment by motorsep** [comment]
@CarlKenner cool, but it's not the same as running it directly on the headset. There will be more latency and that's not desirable for VR at all.

**Comment by cercata** [comment]
I like the idea, I wonder if the GO is powerfull enought for this game ... I'd be happy even with a Doom 1/2 on the GO well done.

**Comment by CarlKenner** [comment]
Yes, I was thinking that Doom 1/2 would definitely work on the GO. But the Doom 1/2 implementation in BFG isn't really suitable for the GO, it's currently just a virtual screen.
